### PR TITLE
 Added description about default.xml

### PR DESCRIPTION
**Pull request by speedy008**
## Purpose of this pull request

This pull request (PR) contains descriotion about default.xml

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/layouts/xml-instructions.html
- https://devdocs.magento.com/guides/v2.2/frontend-dev-guide/layouts/xml-instructions.html
- https://devdocs.magento.com/guides/v2.1/frontend-dev-guide/layouts/xml-instructions.html

whatsnew
Added a new section that describes how to [manage layouts](https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/layouts/xml-instructions.html).

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md [hunk 1]
```diff
@@ -14,6 +14,11 @@ There are two possible ways to customize page layout in Magento:
 
 To change the page wireframe, modify the [page layout] files; all other customizations are performed in the [page configuration] or [generic layout] files. 
 
+In order to add a layout changes that will be available on every page, a module should use deafult.xml
+
+For Example :
+Layout changes that is added to `app/design/frontend/OrangeCo/orange/Magento_Theme/layout/default.xml` will be loaded to all the available page.
```
Comment (rogyar) [inline guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md]: We provide information on how to change layout globally but don't mention a possibility to make changes to particular handles. 
I would suggest adding another sentence about editing local handles. i.e. "To add layout changes to a specific page a module should use layout handle that corresponds the page's path. Example: changes that are added to `app/code/Vendor/Module/view/frontend/layout/catalog_product_view.xml` will be reflected on the product details page"

Comment (rogyar) [reply guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md]: Also, if we are talking about a module, please replace `app/design/frontend/OrangeCo/orange/Magento_Theme/layout/default.xml` with something like `app/code/Vendor/Module/view/frontend/layout/default.xml`

**Comment by rogyar** [comment]
@speedy008. Please note that your change is also applicable for 2.1 version

File: guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md (hunk 1 above)
Comment (speedy008) [reply guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md]: added the changes

File: guides/v2.1/frontend-dev-guide/layouts/xml-instructions.md [hunk 2]
```diff
@@ -13,6 +13,13 @@ There are two possible ways to customize page layout in Magento:
 
 To change the page wireframe, modify the [page layout]({{page.baseurl}}/frontend-dev-guide/layouts/layout-types.html#layout-types-page) files; all other customizations are performed in the [page configuration]({{page.baseurl}}/frontend-dev-guide/layouts/layout-types.html#layout-types-conf) or [generic layout]({{page.baseurl}}/frontend-dev-guide/layouts/layout-types.html#layout-types-gen) files. 
 
+## Layout handle
```
Comment (jeff-matthews) [inline guides/v2.1/frontend-dev-guide/layouts/xml-instructions.md]:
```suggestion
## Manage layouts
```

File: guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md [hunk 3]
```diff
@@ -14,6 +14,13 @@ There are two possible ways to customize page layout in Magento:
 
 To change the page wireframe, modify the [page layout] files; all other customizations are performed in the [page configuration] or [generic layout] files. 
 
+## Layout handle
```
Comment (jeff-matthews) [inline guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md]:
```suggestion
## Manage layouts
```

File: guides/v2.1/frontend-dev-guide/layouts/xml-instructions.md [hunk 4]
```diff
@@ -13,6 +13,13 @@ There are two possible ways to customize page layout in Magento:
...
+
+In order to add a layout changes that will be available on every page, a module should use `deafult.xml` layout.
```
Comment (jeff-matthews) [inline guides/v2.1/frontend-dev-guide/layouts/xml-instructions.md]:
```suggestion
To make layout changes available on every page, modify the `default.xml` file.
```

File: guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md [hunk 5]
```diff
@@ -14,6 +14,13 @@ There are two possible ways to customize page layout in Magento:
...
+
+In order to add a layout changes that will be available on every page, a module should use `deafult.xml` layout.
```
Comment (jeff-matthews) [inline guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md]:
```suggestion
To make layout changes available on every page, modify the `default.xml` file.
```

File: guides/v2.1/frontend-dev-guide/layouts/xml-instructions.md [hunk 6]
```diff
@@ -13,6 +13,13 @@ There are two possible ways to customize page layout in Magento:
...
+For example, layout changes that is added to `app/code/Vendor/Module/view/frontend/layout/default.xml` will be loaded on all pages.
```
Comment (jeff-matthews) [inline guides/v2.1/frontend-dev-guide/layouts/xml-instructions.md]:
```suggestion
For example, layout changes added to `app/code/Vendor/Module/view/frontend/layout/default.xml` are loaded on all pages.
```

File: guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md [hunk 7]
```diff
@@ -14,6 +14,13 @@ There are two possible ways to customize page layout in Magento:
...
+For example, layout changes that is added to `app/code/Vendor/Module/view/frontend/layout/default.xml` will be loaded on all pages.
```
Comment (jeff-matthews) [inline guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md]:
```suggestion
For example, layout changes added to `app/code/Vendor/Module/view/frontend/layout/default.xml` are loaded on all pages.
```

File: guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md [hunk 8]
```diff
@@ -14,6 +14,13 @@ There are two possible ways to customize page layout in Magento:
...
+To add layout changes to a specific page a module should use layout handle that corresponds the page's path. 
+For example, the changes that are added to `app/code/Vendor/Module/view/frontend/layout/catalog_product_view.xml` will be reflected on the product details page.
```
Comment (jeff-matthews) [inline guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md]:
```suggestion
For example, changes to the `app/code/Vendor/Module/view/frontend/layout/catalog_product_view.xml` page are loaded on the product details page.
```

File: guides/v2.1/frontend-dev-guide/layouts/xml-instructions.md [hunk 9]
```diff
@@ -13,6 +13,13 @@ There are two possible ways to customize page layout in Magento:
...
+To add layout changes to a specific page a module should use layout handle that corresponds the page's path. 
+For example, the changes that are added to `app/code/Vendor/Module/view/frontend/layout/catalog_product_view.xml` will be reflected on the product details page.
```
Comment (jeff-matthews) [inline guides/v2.1/frontend-dev-guide/layouts/xml-instructions.md]:
```suggestion
For example, changes to the `app/code/Vendor/Module/view/frontend/layout/catalog_product_view.xml` file are loaded on the product details page.
```

File: guides/v2.1/frontend-dev-guide/layouts/xml-instructions.md [hunk 10]
```diff
@@ -13,6 +13,13 @@ There are two possible ways to customize page layout in Magento:
 
 To change the page wireframe, modify the [page layout]({{page.baseurl}}/frontend-dev-guide/layouts/layout-types.html#layout-types-page) files; all other customizations are performed in the [page configuration]({{page.baseurl}}/frontend-dev-guide/layouts/layout-types.html#layout-types-conf) or [generic layout]({{page.baseurl}}/frontend-dev-guide/layouts/layout-types.html#layout-types-gen) files. 
 
+## Layout handle
+
+In order to add a layout changes that will be available on every page, a module should use `deafult.xml` layout.
+For example, layout changes that is added to `app/code/Vendor/Module/view/frontend/layout/default.xml` will be loaded on all pages.
+To add layout changes to a specific page a module should use layout handle that corresponds the page's path.
```
Comment (jeff-matthews) [inline guides/v2.1/frontend-dev-guide/layouts/xml-instructions.md]: What is a "layout handle"?

File: guides/v2.1/frontend-dev-guide/layouts/xml-instructions.md [hunk 11]
```diff
@@ -13,6 +13,13 @@ There are two possible ways to customize page layout in Magento:
 
 To change the page wireframe, modify the [page layout]({{page.baseurl}}/frontend-dev-guide/layouts/layout-types.html#layout-types-page) files; all other customizations are performed in the [page configuration]({{page.baseurl}}/frontend-dev-guide/layouts/layout-types.html#layout-types-conf) or [generic layout]({{page.baseurl}}/frontend-dev-guide/layouts/layout-types.html#layout-types-gen) files. 
 
+## Manage layouts
+
+To make layout changes available on every page, modify the `default.xml` file.
+For example, layout changes added to `app/code/Vendor/Module/view/frontend/layout/default.xml` are loaded on all pages.
+To add layout changes to a specific page a module should use layout handle that corresponds the page's path.
```
Comment (jeff-matthews) [inline guides/v2.1/frontend-dev-guide/layouts/xml-instructions.md]:
```suggestion
To add layout changes to a specific page, use a layout file that corresponds to the page's path. 
```

File: guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md [hunk 12]
```diff
@@ -14,6 +14,13 @@ There are two possible ways to customize page layout in Magento:
 
 To change the page wireframe, modify the [page layout] files; all other customizations are performed in the [page configuration] or [generic layout] files. 
 
+## Manage layouts
+
+To make layout changes available on every page, modify the `default.xml` file.
+For example, layout changes added to `app/code/Vendor/Module/view/frontend/layout/default.xml` are loaded on all pages.
+To add layout changes to a specific page a module should use layout handle that corresponds the page's path.
```
Comment (jeff-matthews) [inline guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md]:
```suggestion
To add layout changes to a specific page, use a layout file that corresponds to the page's path. 
```

**Comment by jeff-matthews** [comment]
running tests

**Pull request merged**